bftengine/include/bftengine/ReservedPagesClient.hpp

```diff
@@ -94,14 +104,14 @@ class ResPagesClient : public ReservedPagesClientBase, public IReservedPages {
   }
 
   /** is called when calculating absolute pageId */
-  uint32_t my_offset() const {
+  static uint32_t my_offset() {
```

Comment:
my_offset() was intentionally declares private in order not to allow clients to intervene the res pages calc math

Reply:
I will make it private and make `ReservedPagesMock` a friend.